### ISSUE TITLE
add support to reference.docx with panic 

### ISSUE THREAD

**Issue by prtome** [m-rossi/jupyter-docx-bundler#83]
when using pandoc to-docx we can use a template reference.docx that contains the formatting -for example the fonts to be used

It would be nice if this would work with Jupyter-docx-bundler.

**Comment by m-rossi** [comment]
That's already possible as described here: https://github.com/m-rossi/jupyter-docx-bundler/wiki/Custom-export-template

**Comment by prtome** [comment]
Yes, I have my reference.docx file correctly in the data-dir directory, On macOs the directory is : $HOME/.local/share/pandoc

pandoc uses it when running from the command line. But it does not use it when running from nbconvert with jupyter-docx-bundler.

**Comment by m-rossi** [comment]
Oh, it seems like it stopped working. I will look into it.
